### DISCUSSION TITLE
Dishwasher Bosch SMV4HCX48E/24 mqtt payload entries are only available as numbers ?

### DISCUSSION THREAD

**Discussion by zibous** [hcpy2-0/hcpy#78] [Q&A]
I can't figure out why the following entries are only available as numbers: 

***mqtt payload***

- 520: "2024-07-01T09:12:37"
- 535: false
- 547: false
- 4632: 0
- 4641: 0
- 4654: 0
- 
![mqtt_info](https://github.com/hcpy2-0/hcpy/assets/30198737/477f21d5-7c5a-4dca-aa97-180df2d16837)

How can I change this ?

Project see: https://github.com/zibous/homeconnect-hcpy

Thanks

**Comment by Meatballs1** [comment]
The number to name mapping is done via the devices.json pulled from the cloud. 

Not really sure what they represent without this, but doesn't appear to be anything fundamental to control the units

**Reply by zibous** [reply]
Thanks
This would mean that not all entries are there. How can I find out how to enter them in the devices.json

Decices.json see:
https://github.com/zibous/homeconnect-hcpy/blob/main/homeassistant/devices_dishwasher.json

**Reply by Meatballs1** [reply]
Can just manually create the mappings to name them by modifying the JSON if you know what they are and it should automatically pick up whatever name you give them.

But unless.you know what they represent theres not a lot of benefit
